metadata-ingestion/tests/unit/test_sql_common.py

```diff
@@ -105,14 +105,32 @@ def test_get_platform_from_sqlalchemy_uri(uri: str, expected_platform: str) -> N
 
 
 def test_get_db_schema_with_dots_in_view_name():
-    config: SQLCommonConfig = _TestSQLAlchemyConfig()
-    ctx: PipelineContext = PipelineContext(run_id="test_ctx")
-    platform: str = "TEST"
-    source = _TestSQLAlchemySource(config=config, ctx=ctx, platform=platform)
-
+    source = get_test_sql_alchemy_source()
     database, schema = source.get_db_schema(
         dataset_identifier="database.schema.long.view.name1"
     )
-
     assert database == "database"
     assert schema == "schema"
+
+
+def test_test_connection_success():
+    source = get_test_sql_alchemy_source()
+    with mock.patch(
+        "datahub.ingestion.source.sql.sql_common.SQLAlchemySource.get_inspectors",
+        side_effect=lambda: [],
+    ):
+        report = source.test_connection({})
+        assert report is not None
+        assert report.basic_connectivity
+        assert report.basic_connectivity.capable
+        assert report.basic_connectivity.failure_reason is None
+
+
+def test_test_connection_failure():
```

Comment:
this test is great, but can we also add a live test (e.g. without any mocks) to the mysql integration test?